crates/rattler_installs_packages/src/resolve.rs

```diff
@@ -499,6 +566,51 @@ pub struct PinnedPackage<'db> {
     pub artifacts: Vec<&'db ArtifactInfo>,
 }
 
+/// Defines how to handle sdists during resolution.
+#[derive(Default, Clone, Copy, Eq, PartialOrd, PartialEq)]
+pub enum SDistResolution {
+    /// Treat sdists the same as wheels.
```

Comment:
I think this should be worded a bit differently, in the same version `wheels` are preffered but an sdist would be chosen over an older `wheel` version.